src/components/Select/AutosuggestMulti/AutosuggestMulti.tsx

```diff
@@ -39,7 +39,7 @@ export function AutosuggestMulti<S>(props: Props<S>) {
         id,
         onInputValueChange,
         onSelectionChange,
-        selectedSuggestions,
+        selectedSuggestions = [],
```

Comment:
Is adding of this default parameter here intentional? Taking into account it's already defined in `defaultProps` below.

Reply:
yes, otherwise TS complains down the line that it might be undefined